azure-pipelines/end-to-end-tests-dir/asset-caching.ps1

```diff
@@ -69,159 +80,252 @@ if (-not ($actual -match $expected)) {
 # Testing asset cache hit
 Refresh-Downloads
 Run-Vcpkg -TestArgs ($commonArgs + @('remove', 'vcpkg-internal-e2e-test-port'))
-$actual = Run-VcpkgAndCaptureOutput -TestArgs ($commonArgs + @("install", "vcpkg-internal-e2e-test-port", "--overlay-ports=$PSScriptRoot/../e2e-ports", "--x-asset-sources=x-azurl,file://$AssetCache,,readwrite;", "--downloads-root=$DownloadsRoot"))
-$actual = $actual -replace "`r`n", "`n"
+$actual = Run-VcpkgAndCaptureOutput -TestArgs ($commonArgs + @("fetch", "cmake", "--x-asset-sources=x-azurl,file://$AssetCache,,readwrite;", "--downloads-root=$DownloadsRoot"))
 
 $expected = @(
-"A suitable version of .* was not found \(required v[0-9\.]+\)."
-"Asset cache hit for .*; downloaded from: .*"
+"A suitable version of cmake was not found \(required v[0-9\.]+\)\.",
+"Trying to download cmake-[0-9.]+-[^.]+\.(zip|tar\.gz) using asset cache file://$assetCacheRegex/[0-9a-z]+",
+"Download successful! Asset cache hit, did not try authoritative source https://github\.com/Kitware/CMake/releases/download/[^ ]+"
```

Comment:
```suggestion
Asset cache hit!
```

I think that "Asset cache hit!" implies successful download which implies not trying the authoritative source. The previous line says we are engaging with asset cache so I don't think we need to say we didn't try the authoritative source.

Reply:
The original intended URI needs to be in the output somehow. Where do you want it go instead?

Reply:
Why do we need to print the intended URI? In this example, we already say we are looking for cmake and we never actually consult the authoritative source. Why print it?

Reply:
I think it is important for a user to understand what we are doing.

Reply:
I don't believe the file name without the upstream URI(s) is sufficient to describe what we are doing.